api/query/metadata_cache.go

```diff
@@ -21,12 +21,14 @@ func (f searchcacheMetadataFetcher) Fetch() (sha *string, res map[string][]byte,
 		return nil, MetadataMapCached, nil
 	}
 
+	// nolint:exhaustruct // TODO: Fix exhaustruct lint error
```

Comment:
This one still needs to be added to the .yaml file and have this comment removed.